feature/google/src/test/java/BillingWrapperCommonTest.kt

```diff
@@ -168,77 +65,52 @@ class BillingWrapperTest {
     }
 
     @Test
-    fun defersCallingSkuQueryUntilConnected() {
-        mockStandardSkuDetailsResponse()
+    fun defersCallUntilConnected() {
         every { mockClient.isReady } returns false
 
-        val productIDs = setOf("product_a")
-
-        wrapper.querySkuDetailsAsync(
-            ProductType.SUBS,
-            productIDs,
-            {
-                this@BillingWrapperTest.storeProducts = it
-            }, {
-                fail("shouldn't be an error")
-            })
+        val token = "token"
+        var consumePurchaseCompleted = false
+        wrapper.consumePurchase(token) { _, _ ->
+            consumePurchaseCompleted = true
+        }
 
-        assertThat(storeProducts).`as`("SKUDetailsList is null").isNull()
+        assertThat(consumePurchaseCompleted).isFalse
 
         every { mockClient.isReady } returns true
 
         billingClientStateListener!!.onBillingSetupFinished(billingClientOKResult)
 
-        assertThat(storeProducts).`as`("SKUDetailsList is not null").isNotNull
+        assertThat(consumePurchaseCompleted).isTrue
     }
 
     @Test
     fun canDeferMultipleCalls() {
```

Comment:
I know it was like this before but maybe a rename of this test?
```suggestion
    fun canDeferMultipleCallsUntilConnected() {
```